{
  "timestamp": 1539244570333,
  "status": 500,
  "error": "Internal Server Error",
  "exception": "org.springframework.dao.DataIntegrityViolationException",
  "message": "\r\n### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'id' cannot be null\r\n### The error may involve com.zhum.demo.mapper.UserMapper.insert-Inline\r\n### The error occurred while setting parameters\r\n### SQL: insert into t_user (id, userName, password,        phone)     values (?, ?, ?,        ?)\r\n### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'id' cannot be null\n; SQL []; Column 'id' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'id' cannot be null",
  "path": "/bmt/user/add"
}